packages/jest-changed-files/src/git.js

```diff
@@ -15,11 +15,18 @@ import type {Path} from 'types/Config';
 const path = require('path');
 const childProcess = require('child_process');
 
-function findChangedFiles(cwd: string): Promise<Array<Path>> {
+type Options = { 
+  withAncestor?: boolean,
+  lastCommit?: boolean,
+};
+
+function findChangedFiles(cwd: string, options: Options): Promise<Array<Path>> {
   return new Promise((resolve, reject) => {
-    const args = ['ls-files', '--other', '--modified', '--exclude-standard'];
+    let args = ['ls-files', '--other', '--modified', '--exclude-standard'];
     const child = childProcess.spawn('git', args, {cwd});
-
+    if (options.lastCommit === true) {
+      args = ['show', '--name-only', '--pretty=%b', 'HEAD'];
+    }
```

Comment:
can we do:

```js
const args = options.lastCommit
  ? ['show', '--name-only', '--pretty=%b', 'HEAD']
  : ['ls-files', '--other', '--modified', '--exclude-standard'];
```

instead? I prefer not to use `let`. Also, how does this work since you are calling `childProcess.spawn` before you modify args here.